### DISCUSSION TITLE
Regarding the perfusion in Matlab

### DISCUSSION THREAD

**Discussion by Snuag** [ankrh/MCmatlab#19] [Q&A]
the code doesnt consider the effect of perfusion. How do I add perfusion in the code?

**Comment by ankrh** [comment]
Unfortunately there's currently no direct way of accomplishing that in terms of defining the model in MATLAB. I have been thinking about trying to implement the ability to specify perfusion cooling, but I'm not sure when or if I'll get around to implementing it.

If you're up for a challenge you can try to implement it yourself. You would have to define a way to specify which media are cooled by perfusion and at which rate, and then put that into the heat solver by modifying finiteElementHeatPropagator.c (including line 300 for CPU, line 271 for GPU). But I think it will be difficult if you haven't spent a lot of time with the C codebase.
